spec/setup_test_database.rb

```diff
@@ -0,0 +1,7 @@
+require 'pg'
+
+def setup_test_database
+  connection = PG.connect(dbname: 'chitter_test')
```

Comment:
In README, test database is called `chitter_db_test` - might be why this isn't working